keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -207,25 +426,60 @@ promotion is an optional part of this KEP. As an intermediate solution we can
 before graduating the KEP to GA.
 
-All architectures that are supported by the [package building tool][pkg-gen-kep] should be published.
-This KEP suggests to start with publishing a single supported architecture
-(e.g. `linux/amd64`) and extend that iteratively, when we verify that creating
-all packages for all architectures is fast enough to be done as part of the
-release process. If it turns out this step takes too long, we need to think
-about doing the package building & publishing asynchronous to the release
-process (see also: [Risks](#risks-and-mitigations)).
+At the time of writing this KEP, there are no Go libraries for working with OBS that we could use to integrate directly
+with `krel`. Eventually, we could evaluate if it makes sense to build such a library for our purposes. Until then,
+we'll use `osc` directly (by exec-ing), which also requires adding `osc` to our build images.
 
-### Risks and Mitigations
+[obs-rabbitmq]: https://rabbit.opensuse.org/
 
-- _Risk_: We don't find a proper way to share secrets like the signing key
-  _Mitigation_: Using a third party tool like 1Password
-- _Risk_: Building all the packages for all the distributions and their version takes too long to be done nightly or via cutting the release  
-  _Mitigation_: We do not deliver nightly packages.
+#### Authentication to OBS and User Management
 
-## Design Details
+The concept of API tokens in OBS is very limited and provides access only to a very few endpoints. In other words,
+it's not possible to use API tokens for publishing to OBS. Instead, we need to create some sort of a service account
+to be used when publishing packages. This is one time operation that can be done by SIG Release Leads.
+
+Users are managed manually via the OBS web interface. SIG Release Leads must have access to add/remove users from
+our OBS project. Release Managers should be given read/write access, so they can maintain and create projects and
+packages.
+
+#### How are packages used?
+
+The End Users can configure their systems’ package managers to use
+those different `${channel}`s of a kubernetes `${k8s_release}` for their
+corresponding package manager.
+
+A configuration for the package managers might look something like:
+
+- deb (deb822 formatted version):
+  ```
+  Types: deb
+  URIs: http://packages.kubernetes.io/core:/stable:/v1.26/deb/
+  Suites: ./
+  Signed-By: /etc/keyrings/kubernetes-keyring.gpg
+  ```
+- rpm/yum:
+  ```
+  [kubernetes]
+  name=Kubernetes
+  type=rpm-md
+  baseurl=https://packages.kubernetes.io/core:/stable:/v1.26:/build/rpm/
+  gpgcheck=1
+  gpgkey=https://packages.kubernetes.io/core:/stable:/v1.26:/build/rpm/repodata/repomd.xml.key
+  enabled=1
+  ```
+
+**Note**: The (base) URL for packages will be changed from `{apt,yum}.kubernetes.io` to `packages.kubernetes.io`.
+This is to better support the OBS workflow. This is consider as a kind of breaking change, however, since there
```

Comment:
> This is consider as a kind of breaking change

Is there a communication / promotion plan in place for the migration? I understand it is low impact to roll out, but when we _eventually_ cutover, it will become high impact.

Reply:
We plan to come up with a communication plan before graduating this KEP to beta, i.e. after we implement the initial solution. In fact, the communication plan depends on how we implement everything. This is a written down as a beta graduation requirement in the KEP:

> Outline deprecation policy and communication plan